flink-ml-servable-core/src/main/java/org/apache/flink/ml/servable/types/MatrixType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.servable.types;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+/**
+ * A DataType representing a map where all keys have the same BasicType and all values have the same
```

Comment:
The java doc seems wrong here.

Reply:
Ah my bad. Thanks for catching this. It is updated to the following doc now.

```
/** A matrix of double values. */
```